styles.css

```diff
@@ -0,0 +1,236 @@
+* {
+  font-family: 'Open Sans', sans-serif;
```

Comment:
I'm not a huge fan of declaring a master font for everything. I generally don't put much in this * selector than box-sizing: border-box;